wire-library/wire-runtime/src/jvmMain/kotlin/com/squareup/wire/internal/FieldBinding.kt

```diff
@@ -100,6 +103,15 @@ class FieldBinding<M : Message<M, B>, B : Message.Builder<M, B>> internal constr
     }
   }
 
+  private fun getInstanceGetter(messageType: Class<M>): (M)->Any? =
+    if (Modifier.isPrivate(messageField.modifiers)) {
+      val getterName = "get" + messageField.name.replaceFirstChar { it.uppercase() }
+      val getter: Method = messageType.getMethod(getterName)
+      ({ instance -> getter.invoke(instance) })
```

Comment:
```suggestion
      { instance -> getter.invoke(instance) }
```

Reply:
might as well be able to fancy return the reference....

Reply:
Yeah, I don't get it either